### PR TITLE
Change EIP-2982 status from "Draft" to "Review"

### PR DESCRIPTION
**Pull request by poojaranjan**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2982 state from Draft to Review
 - EIP 2982 requires approval from one of (@djrtwo, @vbuterin)

**Comment by MicahZoltu** [comment]
I'm still against this being an EIP, but I believe I have lost that battle I believ.  I'll leave it to one of the other editors to review and merge it.  I recommend at least getting it in line with our general standards process (no external links, no non-standard sections, short simple summary, minimal opinion/argumentation, purely technical document, etc.).

**Review by lightclient — APPROVED** [review]

**Pull request merged**
